DataConnectors/Okta Single Sign-On/azuredeploy_OktaSingleSignOn_API_FunctionApp_V2.json

```diff
@@ -204,19 +210,6 @@
                    "alwaysOn": true
               }
           },
-          {
```

Comment:
This will create a Web App and expose it through Private Endpoint. 
Parameterize "name": "[concat(variables('FunctionName'), '/', variables('FunctionName'), parameters('webapp_dns_name'))]",
if environment.portal is .com use .net
else use the extension like (.us)

Reply:
Sreedhar, we remove this dns settings purposely, because this is an azure function working as a worker role and it doesn't expose any endpoint. Plus, the code is adding the default host name binding which is in place. We are also planning to remove this in the template. Do you know any previous reason why this is added in the first place?

Reply:
No change is needed here, hence resolving.